src/appleseed-max-impl/appleseedrenderer/projectbuilder.cpp

```diff
@@ -856,12 +870,15 @@ namespace
         const RendererSettings& settings,
         const TimeValue         time,
         ObjectMap&              object_map,
+        ObjectInstanceMap&      obj_instance_map,
         MaterialMap&            material_map,
         AssemblyMap&            assembly_map)
     {
         // Retrieve the geometrical object referenced by this node.
         Object* object = node->GetObjectRef();
 
+        std::map<asr::ObjectInstance*, ObjectInfo> instance_map;
```

Comment:
What's the difference between `obj_instance_map` (passed in argument) and `instance_map`?